From b93319dde7f8f35e062b67eb28826babcd183cea Mon Sep 17 00:00:00 2001
From: Feng Pei <quic_fpei@quicinc.com>
Date: Fri, 21 Jun 2024 01:23:19 -0700
Subject: [PATCH] Revert "net: disable Werror temporarily"

This reverts commit 26645ebfb0d8a7b54acfb4ee6be15cf14e434e20.

Reason for revert: <warning error has been fixed >

Change-Id: Id27d25d063cd20d3d97dfee811d4723f1fce7a98
---
 net/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/net/Makefile b/net/Makefile
index 26d8d19fed79..eac547aca3b4 100644
--- a/net/Makefile
+++ b/net/Makefile
@@ -5,7 +5,7 @@
 # 2 Sep 2000, Christoph Hellwig <hch@infradead.org>
 # Rewritten to use lists instead of if-statements.
 #
-KBUILD_CPPFLAGS := $(filter-out -Werror, $(KBUILD_CPPFLAGS))
+
 obj-y				:= devres.o socket.o core/
 
 obj-$(CONFIG_COMPAT)		+= compat.o
-- 
2.34.1

